jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/RetrieveRegistryCredentialsStep.java

```diff
@@ -84,8 +84,6 @@ static RetrieveRegistryCredentialsStep forTargetImage(
         }
       }
 
-      // If no credentials found, give an info (not warning because in most cases, the base image is
-      // public and does not need extra credentials) and return empty.
```

Comment:
should we leave this comment in?

Reply:
I intend to remove this. Doesn't seem to help much. For example, when looking at this line, you will not question why this is `info()` instead of `warning()`.